lib/cylc/gui/updater_graph.py

```diff
@@ -235,6 +236,8 @@ def run(self):
             sleep(0.2)
 
     def update_xdot(self, no_zoom=False):
+        colors = getattr(self.xdot.widget.style, 'bg', None)
+        self.graphw.graph_attr['bgcolor'] = colors[gtk.STATE_NORMAL]
         self.xdot.set_dotcode(self.graphw.to_string(), no_zoom=no_zoom)
```

Comment:
(note that it seems not to be possible to get theme colors before a window is "realized" - so I couldn't do it as a default during graph definition)